### DISCUSSION TITLE
blog/2024/05/10/new-touching-fish

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#1] [General]
# blog/2024/05/10/new-touching-fish

今日不更新

http://localhost:8080/blog/2024/05/10/new-touching-fish

**Comment by MingGH** [comment]
怎么可能不更新！！至少增加了评论系统，但是我还没有找到怎么在摸鱼忍术中增加评论组件，前端好难

**Comment by MingGH** [comment]
难道我真的是个天才？还是被我摸索出来了，等我发版一波

**Comment by MingGH** [comment]
1
